src/librustc_error_codes/error_codes/E0724.md

```diff
@@ -1,4 +1,5 @@
-`#[ffi_returns_twice]` was used on non-foreign function.
+`#[ffi_returns_twice]` was used on something else than a foreign function
```

Comment:
```suggestion
`#[ffi_returns_twice]` was used on something other than a foreign function
```
Is this more clear than before? What happens if you try and put this over a struct or something, is it the same error?